src/reset-password/service.ts

```diff
@@ -17,22 +18,28 @@ export async function sendResetPasswordEmail(user: User) {
     }
 
     const transporter = nodemailer.createTransport(process.env.SMTP_URL);
-
+    const token = await createToken(user);
+    const emailTemplate =
+    render('emails/reset-password-email', {
+        name: user.nickname,
+        url: 'https://auth-server.example/reset-password/token/' + token,
```

Comment:
Url is definitely wrong right now... We can fix this in a future PR... not 100% sure how to do this yet.